letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if (integer >= 90)
+    return "A"
+  elsif (integer >= 80)
+    return "B"
+  elsif (integer >= 70)
+    return "C"
+  elsif (integer >= 60)
+    return "D"
+  else
+    return "F"
+  end
```

Comment:
Note that Ruby doesn't require the parenthesis or `return` keywords in each `if`/`elsif` section, so you can remove them.

The return keyword isn't needed because the function's return value is implicitly set to the last statement (which is the entire `if`/`elsif` block and because each section of the block also returns the last statement (each string).